packages/ses/test/import.test.js

```diff
@@ -600,3 +601,21 @@ test('importMetaHook and meta from record', async t => {
   const { default: metaurl } = await compartment.import('./index.js');
   t.is(metaurl, 'https://example.com/index.js?foo');
 });
+
+test('dynamic import from source', async t => {
+  const c = new Compartment({
+    __options__: true,
+    __noNamespaceBox__: true,
+    resolveHook: s => s,
+    modules: {
+      '-': {
```

Comment:
Does this have a specific meaning, or could it have been `foo`?

Reply:
It could have been `'foo'`, `'.'`, or even `''`.